app/scripts/controllers/ModalCtrl.js

```diff
@@ -0,0 +1,23 @@
+(function() {
+  function ModalCtrl($uibModal, Room) {
+
+    this.open = function() {
+      var modalInstance = $uibModal.open({
+        templateUrl: '/templates/modal.html',
+        controller: 'ModalInstanceCtrl',
+        controllerAs: 'modal',
```

Comment:
If you wanted to, you could put this all on one line like so:
`controller: 'ModalInstanceCtrl as modal'
But it's not necessary. Just wanted to let you know the option ;)